foundation-extensions/mobile-core/configuration/README.md

```diff
@@ -263,7 +263,41 @@ ACPCore.UpdateConfiguration(config);
 {% endtab %}
 {% endtabs %}
 
-## Clearing programmatic updates to the configuration (iOS 3.x.x Only)
+## Clearing programmatic updates to the configuration (Android and iOS (AEP 3.x) Only)
```

Comment:
@spoorthipujariadobe any strong reason to keep this on the extension home page or would the API reference be sufficient for this example?

Reply:
The section above this one in the README has details on programmatically updating the configuration. I'm guessing that's why this section was added right below it. @cdhoffmann any other reason I'm missing?